README.md

```diff
@@ -180,3 +185,6 @@ The following table lists the configurable parameters of this chart and their de
 | `postgres.dataMountPath`             | Path for Postgres data storage                  | `nil`                                                         |
 | `affinity`                 | Affinity settings for pod assignment            | {}                                                         |
 | `tolerations`              | Toleration labels for pod assignment            | []
+
+## Performance
+We have observed that gitea performance is heavily file system dependent. If high performance is required be sure to use fast storage. Otherwise tune each container resource settings to suit your needs.
```

Comment:
```suggestion
We have observed that gitea performance is heavily file system dependent. If high performance is required be sure to use fast storage. Otherwise tune container resource settings to suit your needs.
```